generators/app/templates/js/store/configureStore.js

```diff
@@ -6,13 +6,12 @@ let createStoreWithMiddleware;
 
 // Configure the dev tools when in DEV mode
 if (__DEV__) {
-  let {devTools, persistState} = require('redux-devtools');
+  const {devTools, persistState} = require('redux-devtools');
```

Comment:
There are a whole bunch of lint errors, I was going to submit them in a separate PR